client/src/controllers/MessagesController.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class MessagesController extends Controller<HTMLFormElement> {
+  static classes = ['newMessage', 'show'];
+  static targets = ['messages', 'templates'];
+
+  declare readonly hasMessagesTarget: boolean;
+  declare readonly hasNewMessageClass: boolean;
+  declare readonly hasShowClass: boolean;
+  declare readonly hasTemplatesTarget: boolean;
+  declare readonly messagesTarget: HTMLElement;
+  declare readonly newMessageClass: string;
+  declare readonly showClass: string;
+  declare readonly templatesTarget: HTMLElement;
+
+  add(event) {
+    const status = event.detail.status;
+    const statusText = event.detail.text;
+    // console.log("connected add.",status,statusText);
+    // console.log(this.templatesTarget);
+    if (this.hasNewMessageClass) {
+      this.element.classList.add(this.newMessageClass);
+    }
+    if (this.hasTemplatesTarget) {
+      const template = this.templatesTarget as HTMLTemplateElement;
+      const cloneTemplate = template.content.cloneNode(true) as HTMLElement;
+      const listElement = cloneTemplate.querySelector(
+        '[data-message-status="' + status + '"]',
+      ) as HTMLElement;
+      if (listElement) {
+        const spanElement = listElement.querySelector('span') as HTMLElement;
+        spanElement.textContent = statusText;
+        if (this.hasMessagesTarget) {
+          // console.log(listElement);
+          this.messagesTarget.appendChild(listElement);
+        }
+      }
+      const addMessageTimeout = setTimeout(() => {
+        if (this.hasShowClass) {
+          this.element.classList.add(this.showClass);
+        }
+        clearTimeout(addMessageTimeout);
+      }, 100);
```

Comment:
Let's put this in a Stimulus value, something like `showDelay`, a Number that defaults to 100.

```suggestion
      }, this.showDelayValue);
```

It's a small change now and does not impact the initial DOM attributes but allows for more customisation of this behaviour down the road if needed.